front-end/src/components/Container/MainContainer/MainContainer.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { StyledContainer } from './styled';
+import { propTypes, defaultProps } from './props';
+
+export const MainContainer = ({ children, ...rest }) => {
+  return <StyledContainer {...rest}>{children}</StyledContainer>;
```

Comment:
check spacing on both sides of {}

Reply:
This spacing is being managed by prettier. On save it formats according to this style.